app/models/institution.rb

```diff
@@ -210,6 +210,10 @@ def school?
     institution_type_name != 'OJT'
   end
 
+  def programs
+    Program.where('facility_code' => self.facility_code)
```

Comment:
Code climate says self is redundant.

Reply:
Changes are in. Github just took it's time showing the update.